assets/src/components/loggedInAs.tsx

```diff
@@ -1,8 +1,15 @@
-import React from "react"
+import React, { ComponentPropsWithoutRef } from "react"
 import { UserAvatar } from "./userAvatar"
+import { joinClasses } from "../helpers/dom"
 
-export const LoggedInAs = ({ email }: { email: string }) => (
-  <div className="c-logged-in-as__container">
+export const LoggedInAs = ({
+  email,
+  ...divProps
+}: { email: string } & ComponentPropsWithoutRef<"div">) => (
+  <div
+    {...divProps}
+    className={joinClasses(["c-logged-in-as__container", divProps.className])}
+  >
```

Comment:
@mbta/skate-developers what do you think about this approach to adding the classes needed for padding?

I _could_ wrap this in a `<div/>` in `<NavMenu/>`, but where possible, I prefer to not introduce more `div`'s if it can be avoided.

Reply:
I'm still new enough to React-land that if this is somehow evil, I wouldn't necessarily catch it, but I really like the effect this has, of being able to add classes (and other attributes if needed) to the outer div here, so I'm all for it.

Reply:
This _seems_ to be fairly normal in React (see, react-bootstrap making basically every HTMLElement attribute available, and more available via `to={}`) But I'm not sure where it's too much, and if this is even the right way to go about it, but I think it's fine so far.